packages/core-components/src/components/Status/Status.tsx

```diff
@@ -32,123 +38,187 @@ const useStyles = makeStyles(
-      },
+      fill: theme.palette.status.aborted || '#6A6E73',
     },
   }),
   { name: 'BackstageStatus' },
 );
 
 export function StatusOK(props: PropsWithChildren<{}>) {
-  const classes = useStyles(props);
+  const { children, ...otherProps } = props;
+  const classes = useStyles(otherProps);
   return (
     <Typography
       component="span"
-      className={classNames(classes.status, classes.ok)}
+      className={classNames(classes.status)}
       aria-label="Status ok"
       aria-hidden="true"
-      {...props}
-    />
+      {...otherProps}
+    >
+      <CheckCircleOutline
+        data-testid="status-ok"
+        className={classNames(
+          classes.ok,
+          classes.statusIconSize,
+          classes.statusIcon,
+        )}
+      />
+      {children}
+    </Typography>
   );
 }
 
 export function StatusWarning(props: PropsWithChildren<{}>) {
-  const classes = useStyles(props);
+  const { children, ...otherProps } = props;
+  const classes = useStyles(otherProps);
   return (
     <Typography
       component="span"
-      className={classNames(classes.status, classes.warning)}
+      className={classNames(classes.status)}
       aria-label="Status warning"
       aria-hidden="true"
-      {...props}
-    />
+      {...otherProps}
+    >
+      <WarningOutline
+        data-testid="status-warning"
+        className={classNames(
+          classes.warning,
+          classes.statusIconSize,
+          classes.statusIcon,
+        )}
+      />
+      {children}
+    </Typography>
   );
 }
 
 export function StatusError(props: PropsWithChildren<{}>) {
-  const classes = useStyles(props);
+  const { children, ...otherProps } = props;
+  const classes = useStyles(otherProps);
   return (
     <Typography
       component="span"
-      className={classNames(classes.status, classes.error)}
+      className={classNames(classes.status)}
       aria-label="Status error"
       aria-hidden="true"
-      {...props}
-    />
+      {...otherProps}
+    >
+      <ErrorOutline
+        data-testid="status-error"
+        className={classNames(
+          classes.error,
+          classes.statusIconSize,
+          classes.statusIcon,
+        )}
+      />
+      {children}
+    </Typography>
   );
 }
 
 export function StatusPending(props: PropsWithChildren<{}>) {
-  const classes = useStyles(props);
+  const { children, ...otherProps } = props;
+  const classes = useStyles(otherProps);
   return (
     <Typography
       component="span"
-      className={classNames(classes.status, classes.pending)}
+      className={classNames(classes.status)}
       aria-label="Status pending"
       aria-hidden="true"
-      {...props}
-    />
+      {...otherProps}
+    >
+      <PendingIcon
+        dataTestId="status-pending"
+        className={classNames(
+          classes.pending,
+          classes.statusIconSizeForImg,
+          classes.statusIcon,
+        )}
+        aria-label="Status pending"
```

Comment:
This is a duplicate of the one that's on the typography - this should be removed right? Unless it's the case that ALL aria labels actually should be only on the icon.

Reply:
Good catch ! I overlooked removing that.